Dockerfile

```diff
@@ -110,6 +110,7 @@ RUN pip install scipy
 RUN pip install git+https://github.com/chanzuckerberg/srst2
 # TODO: Test both pip installations, consider keeping pip use consistent
 RUN pip3 install pandas
+RUN pip3 install psutil==5.6.3
```

Comment:
I want to move the dependencies into a single file but I don't want to do that as part of this PR. Too many moving parts.